packages/dashboard/src/app/views/templateDetails/content/index.js

```diff
@@ -42,19 +41,6 @@ function Content({
   template,
   templateActions,
 }) {
-  const previousTemplateId = useRef(template?.id);
-
-  const {
-    actions: { scrollToTop },
-  } = useLayoutContext();
-
-  useEffect(() => {
-    if (template !== null && template?.id !== previousTemplateId.current) {
-      scrollToTop();
-      previousTemplateId.current = template.id;
-    }
-  }, [template, scrollToTop]);
```

Comment:
This runs every time the template was changed. Now `scrollToTop` is only called if you click `see details` on a related template